### DISCUSSION TITLE
Incorporate backbone NMR chemical shifts in peptide structure minimization protocol in pyRosetta

### DISCUSSION THREAD

**Discussion by rkgasn** [RosettaCommons/rosetta#371] [PyRosetta] [answered]
I have created a denovo structure of a peptide in pyRosetta which includes nCAA. I was wondering if it is possible to incorporate just the experimental backbone NMR chemical shifts into the structure minimization protocol in pyRosetta ? Please let me know.

**Comment by dgront** [comment]
Hi,

one option would be to predict Phi/Psi angles based on NMR chemical shifts
and then use this Phi/Psi angles as restraints / energy function.

Historically, NMR chemical shifts were used to bias fragment generation,
but I'm not sure if this applies to your case.

Best regards,
Dominik Gront

pon., 24 mar 2025 o 22:53 Rittik Ghosh ***@***.***>
napisał(a):

**Reply by rkgasn** [reply]
Hello Dominik,

Many thanks for your answer, I will give this idea a try. So, if I understand correctly, I would use psipred to predict the Phi/Psi angle from the NMR chemical shifts and use them as restraints during the structure minimization. The only issue here could be the nCAA in the sequence. Can psipred handle non-canonical amino acids ? 

Thanks
Rittik K Ghosh

**Comment by dgront** [comment, marked as answer]
Dear Rittik,

I'm not sure if PsiPred predicts Phi/Psi dihedrals. I knew it from it's
high accuracy secondary structure prediction. There were many tools
designed for this purpose published last two decades, e.g. Spatra/Talos or
PREDITOR, but they apparently are no longer maintained. After a quick
search I found this one:
https://www.mdpi.com/1420-3049/28/20/7046

I haven't tried that code, but it's at least a staring point.

Also, you should see the larger picture in mind. Has your peptide well
defined secondary structure? If so, you can use that as a constrain. If
not, the predicted dihedrals may be incorrect. The tools for Phi/Psi
prediction are just neural networks trained on globular proteins. They
assume your case is a medium size protein with helices and sheets, because
this was used as the training data.

>The only issue here could be the nCAA in the sequence.
Phi/Psi predictors are quite inaccurate, I wouldn't bother with that; at
least not in the beginning.

Best,
Dominik Gront

wt., 25 mar 2025 o 13:40 Rittik Ghosh ***@***.***> napisał(a):

**Comment by rkgasn** [comment]
Dear Dominik,

Thank you for your detailed response and suggestions. Based on the 15N NMR chemical shifts data, my peptide appears to be predominantly helical with some sections of random coil. Taking your advice into consideration, I am currently utilizing these chemical shifts as constraints for the peptide model. This approach should yield more accurate results. For now, keeping the nCAA out of the constraints.

Thanks
Rittik
